src/server/rest/v1/service/UserService.ts

```diff
@@ -808,6 +808,7 @@ export default class UserService {
         userIDs: (filteredRequest.UserID ? filteredRequest.UserID.split('|') : null),
         roles: (filteredRequest.Role ? filteredRequest.Role.split('|') : null),
         statuses: (filteredRequest.Status ? filteredRequest.Status.split('|') : null),
+        technical: filteredRequest.Technical ? Utils.convertToBoolean(filteredRequest.Technical) : null,
```

Comment:
y'en a qui mettent des parenthèses, je sais pas pourquoi ça ne sert bien à rien on est d'accord ?
Secondo: calling Utils.isBoolean on a string is clearly not necessary...